RecoJets/Configuration/python/RecoJets_EventContent_cff.py

```diff
@@ -145,38 +103,35 @@
+                                                                     'keep *_hiFJRhoProducer_*_*'
+                                                                     ])
+                )
 #HI-specific products: needed in AOD, propagate to more inclusive tiers as well
-for ec in [RecoJetsAOD.outputCommands, RecoJetsRECO.outputCommands, RecoJetsFEVT.outputCommands]:
-    peripheralPbPb.toModify( ec, 
-                             func=lambda outputCommands: outputCommands.extend(['keep recoCentrality*_pACentrality_*_*'])
-                             )
-
-for ec in [RecoJetsAOD.outputCommands, RecoJetsRECO.outputCommands, RecoJetsFEVT.outputCommands]:
-    pp_on_AA_2018.toModify( ec, 
-                                   func=lambda outputCommands: outputCommands.extend(['keep *_hiCentrality_*_*',
-                                                                                      'keep *_hiFJRhoProducer_*_*',
-                                                                                      'keep *_akPu3PFJets_*_*',
-                                                                                      'keep *_akPu4PFJets_*_*',
-                                                                                      'keep *_kt4PFJetsForRho_*_*',
-                                                                                      'keep *_akCs4PFJets_*_*',
-                                                                                      'keep *_akPu4CaloJets_*_*',
-                                                                                      'drop *_caloTowers_*_*'
+peripheralPbPb.toModify( RecoJetsAOD.outputCommands, 
+                         func=lambda outputCommands: outputCommands.extend(['keep recoCentrality*_pACentrality_*_*'])
+                         )
 
-                                                                         ])
-    )    
+pp_on_AA_2018.toModify( RecoJetsAOD.outputCommands, 
+                        func=lambda outputCommands: outputCommands.extend(['keep *_hiCentrality_*_*',
+                                                                           'keep *_hiFJRhoProducer_*_*',
+                                                                           'keep *_akPu3PFJets_*_*',
+                                                                           'keep *_akPu4PFJets_*_*',
+                                                                           'keep *_kt4PFJetsForRho_*_*',
+                                                                           'keep *_akCs4PFJets_*_*',
+                                                                           'keep *_akPu4CaloJets_*_*',
+                                                                           'drop *_caloTowers_*_*'
+                                                                           ])
+                        )
+RecoJetsRECO.outputCommands.extend(RecoJetsAOD.outputCommands)
```

Comment:
To improve the clarity, I would add these ```extend``` commands right after the definition of the corresponding output commands above, so that it will become more evident that the RecoEventContent (e.g.) is an extention of the AOD event content, and similarly the FEVT event content is an extention of the Reco one